gateways/istio-ingress/templates/deployment.yaml

```diff
@@ -92,18 +96,23 @@ spec:
           - router
           - --domain
           - $(POD_NAMESPACE).svc.{{ .Values.global.proxy.clusterDomain }}
-          - --log_output_level
-          - '{{ $gateway.debug | default "info" }}'
-          - --proxyLogLevel
-          - '{{ $gateway.debug | default "info" }}'
+        {{- if .Values.global.proxy.logLevel }}
+          - --proxyLogLevel={{ .Values.global.proxy.logLevel }}
+        {{- end}}
+        {{- if .Values.global.proxy.componentLogLevel }}
+          - --proxyComponentLogLevel={{ .Values.global.proxy.componentLogLevel }}
+        {{- end}}
+        {{- if .Values.global.logging.level }}
+          - --log_output_level={{ .Values.global.logging.level }}
+        {{- end}}
           - --drainDuration
-          - '{{ $gateway.drainDuration }}'
+          - '45s' #drainDuration
```

Comment:
Can you add back the option ? For ingress it's important to be able to customize this.
Same for connect timeout